default-config.xml

```diff
@@ -66,6 +66,8 @@
         <property name="userAttributes">
             <map>
                 <!--This is now required because the parallelJobsNumber setting is not entirely reliable-->
+		    <!-- This is how node workingDir could be moved to different partition. -->
+				<entry key="ggComputeWorkingDir" value="W:\DCWorkingDir" />
```

Comment:
To make a file easier to read it is good to respect the indenting - can you fix the indents here to respect the proper indent level.